moveit_cartesian_plan_plugin/include/moveit_cartesian_plan_plugin/add_way_point.h

```diff
@@ -75,13 +75,13 @@ Q_OBJECT
 
 	//! Fucntion for all the interactive marker interactions
 	virtual void processFeedback( const visualization_msgs::InteractiveMarkerFeedbackConstPtr &feedback );
-    
+
     //! Make a new Interactive Marker Way-Point
 	virtual void makeArrow(const tf::Transform& point_pos,int count_arrow);
 	//! User Interaction Arrow Marker
 	virtual void makeInteractiveMarker();
 	//! This function listens to the Mouse Interaction Publish Point Topic
-	virtual void msgCallback(const boost::shared_ptr<const geometry_msgs::PointStamped>& point_ptr);
+	// virtual void msgCallback(const boost::shared_ptr<const geometry_msgs::PointStamped>& point_ptr);
```

Comment:
Is this no longer needed?  It's a public method, should we keep it in there for backwards compatibility(the answer could be no)?